setup.json

```diff
@@ -93,6 +93,6 @@
 	"pymatgen>=2019.7.2,<=2022.02.03,!=2019.9.7",
 	"lxml",
 	"packaging",
-	"parsevasp@git+https://github.com/aiida-vasp/parsevasp@058bb26872588c4a83dbc93afcfbf638963be1fc"
+	"parsevasp@git+https://github.com/aiida-vasp/parsevasp@develop"
```

Comment:
I think we should still use a specific know-to-work version as otherwise new change in parsevasp can break the pulgin.

Reply:
Okey, good. Yes, totally agree. Forgot to change this. Will pin it to current develop. After that is submitted I will merge this PR.